src/components/Menu/Menu.test.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import Menu from './Menu.component';
+import Theme from '../App/App.styled';
+
+describe('Menu Component Tests', () => {
+  const setup = () => {
+    const utils = render(
+      <BrowserRouter>
+        <Theme>
+          <Menu />
+        </Theme>
+      </BrowserRouter>
+    );
+    return {
+      ...utils,
+    };
+  };
+
+  it('Should render the menu component', () => {
+    const { getByTestId, asFragment } = setup();
+    expect(getByTestId('CustomAppBar')).toBeInTheDocument();
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('Should redirect to home', async () => {
+    const { getByTestId } = setup();
+    const menuButton = getByTestId('MenuButton');
+    menuButton.click();
+    /* TODO */
```

Comment:
I believe you're adding the test on the next deliverable 😊

Reply:
I wanted to ask you for advice on how should I test this? Because when the button is clicked It redirects to home but because I'm only rendering the menu it doesn't exist, I thought about a spy but I would need to extract the redirect function, any recommendation? 😅

Reply:
I think I can test this functionality on the new `Routes` component I created because it contains the homepage 😄

Reply:
I finally took this test to an upper scope, to the `App` component, because there I have access to the menu bar to do the click on the home button and see the page redirection.